lib/better_errors/middleware.rb

```diff
@@ -33,7 +33,7 @@ class Middleware
     # Adds an address to the set of IP addresses allowed to access Better
     # Errors.
     def self.allow_ip!(addr)
-      ALLOWED_IPS << IPAddr.new(addr)
+      ALLOWED_IPS << (addr.is_a?(IPAddr) ? addr : IPAddr.new(addr))
```

Comment:
What about to make it more safe? (code not tested)

```ruby
case addr
  when String then IPAddr.new(addr)
  when IPAddr then addr
  else raise "Unknown IP address type. Please specify it as a string like "1.1.1.1" or pass IPAddr object."
end
```

Reply:
IMHO it doesn't add much since IPAddr will already raise something like `IPAddr::AddressFamilyError (address family must be specified)` if you pass invalid input.  I can add it if you prefer, though.

Reply:
I agree, it seems like the error raised by `IPAddr` would be sufficient. But also, this PR doesn't change the way that this error would be raised, currently it doesn't rescue from that error.

Reply:
I wasn't able to work past the error in my case without using the wrapped object.  Something not liked about my CIDR string, '172.16.0.0/12' is perfectly valid CIDR range though, which can be parsed from web_console.allow_whitelisted_ips for example

I was able to work past it with #444, thanks for merging this, my report is in #451